Quick and dirty hack to allow reinstalling python packages in an 
existing build. 

Upstream: n/a (local customization)

Signed-off-by: Andreas Ziegler <15275159+aeolio@users.noreply.github.com>
---
--- a/src/installer/destinations.py	2026-05-11 20:12:57.500843800 +0200
+++ b/src/installer/destinations.py	2026-10-02 06:59:57.189404164 +0200
@@ -132,7 +132,7 @@
     this was the root.
     """
 
-    overwrite_existing: bool = False
+    overwrite_existing: bool = True
     """Silently overwrite existing files."""
 
     def _path_with_destdir(self, scheme: Scheme, path: str) -> Path:
